src/components/TableCard/_table-card.scss

```
@@ -27,7 +27,7 @@
 
   p {
     margin-bottom: 8px;
-    font-size: 14px;
+    font-size: carbon--type-scale(1);
```

In this case since we're specifically deviating on only a few properties we can still use the type scale function, although the type scale isn't zero indexed - it starts at `1` ([source](https://github.com/carbon-design-system/carbon/blob/20d37e50319e01ecd9f35da2ce83b6db41b0f088/packages/type/scss/_scale.scss#L28)).  

```suggestion
    font-size: carbon--type-scale(2);
```